app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -140,6 +140,9 @@ class HomeFragment : Fragment() {
         }
 
         override fun onCollectionRenamed(tabCollection: TabCollection, title: String) {
+            lifecycleScope.launch(Main) {
```

Comment:
nit - let's use `viewLifecycleOwner.lifecycleScope.launch` to match the lifecycle of the fragment view